driver-core/src/test/unit/com/mongodb/internal/connection/DnsMultiServerClusterSpecification.groovy

```diff
@@ -70,6 +74,24 @@ class DnsMultiServerClusterSpecification extends Specification {
         initializer != null
         1 * dnsSrvRecordMonitor.start()
 
+        when: 'the current description is accessed before initialization'
```

Comment:
Now that I think of it, these tests are more properly BaseCluster tests.  I put them here because that was where the bug was found.  

Let me know if you think they should be moved.

Reply:
Since this tests setting the description in the `BaseCluster` class, the test should go in the BaseCluster tests.

Reply:
Done